### PR TITLE
Helical Tools Guide Started

### PR DESCRIPTION
**Pull request by BlaviButcher**
Added geno analyse - more or less putting this PR up to check I've got the right idea

### CODE REVIEW

**Comment by jhwz** [comment]
The files should be in the static directory of the project, not the content. Maybe create a subdirectory like `static/geno/...`

File: content/learn/helical-tools/geno.md [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+pcx-content-type: tutorial
+weight: 2
+meta:
+  title: Geno
+---
+
+# Geno
+
+## Getting Started
+
```
Comment (jhwz) [inline content/learn/helical-tools/geno.md:12]: Add a little blurb up here explaining what the tutorial will do, and that the files should be downloaded to follow along with the tutorial

File: content/learn/helical-tools/geno.md [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+### genotype File
+
+A file that holds the genotype data for animals. It is arranged in a matrix where each row represents a different animal and each column represents a different marker (SNP). The first column will be an identifier for the animal.
+
+<a href="../../../downloads/genos" download>Here</a> is the sample geno file we will be using for this tutorial.
```
Comment (jhwz) [inline content/learn/helical-tools/geno.md]: Make sure to update these links to be root relative, these kind of assets should be in the static directory

File: content/learn/helical-tools/geno.md [hunk 3]
```diff
@@ -0,0 +1,175 @@
...
+Pairs with a .geno file and holds information about the markers. The first column is an identifier for the marker. The second column is the chromosome number. The third column is the position on the chromosome.
+
+<a href="../../../downloads/genos.map" download>Here</a> is the sample map file we will be using for this tutorial.
+
+## Analyse
+
+With the geno file we may want a summary of the data. This can be done with the `analyse` command.
+
+```sh
+$ helical geno analyse path/to/genos/file
```
Comment (jhwz) [inline content/learn/helical-tools/geno.md:30]: Add a note saying you are going to assume the genotypes have been downloaded a file called `genotypes`. Makes it more readable

File: content/learn/helical-tools/geno.md [hunk 4]
```diff
@@ -0,0 +1,175 @@
...
+| SNP | COUNT | PROP | GENO MEAN | GENO STDDEV | SNP MEAN | SNP STDDEV |
++-----+-------+------+-----------+-------------+----------+------------+
+| AA  | 246   | 0.25 | 12.30     | 3.53        | 4.92     | 1.89       |
+| AB  | 244   | 0.24 | 12.20     | 3.64        | 4.88     | 1.59       |
+| BB  | 277   | 0.28 | 13.85     | 3.45        | 5.54     | 2.12       |
+| NC  | 233   | 0.23 | 11.65     | 3.12        | 4.66     | 1.88       |
++-----+-------+------+-----------+-------------+----------+------------+
+```
+
+- Snps: signifies how many snps are found for each individual.
```
Comment (jhwz) [inline content/learn/helical-tools/geno.md:52]:
```suggestion
- SNPs: signifies how many snps are found for each individual.
```

File: content/learn/helical-tools/geno.md [hunk 5]
```diff
@@ -0,0 +1,175 @@
...
+- Average MAF: the average minor allele frequency.
+- Markers: the total amount of snps in the file.
+
+Then there is a summary of each potential combination of alleles (AA, AB, BB, NC(not counted)).
```
Comment (jhwz) [inline content/learn/helical-tools/geno.md:56]:
```suggestion
Then there is a summary of each potential combination of alleles (AA, AB, BB, NC(No Call)).
```
Remember that this is no call? We talked about it the other day! Try write this stuff down, or even better create pages on the docs site about it :)

File: content/learn/helical-tools/geno.md [hunk 6]
```diff
@@ -0,0 +1,175 @@
...
+For each combination there is a geno mean which signifies the average of that given allele found in a row/individual. There is also a snp mean which is the average of all the snps found in a column/snp.
+
+If you want more details you can specifify a directory to store additional files. Lets store them in your Documents folder.
```
Comment (jhwz) [inline content/learn/helical-tools/geno.md:59]: Maybe best to just create a temporary folder in the current working directory

File: content/learn/helical-tools/geno.md [hunk 7]
```diff
@@ -0,0 +1,175 @@
...
+
+This will yield the output we want, but it'll be in poor format. This is a good example of how `column` comes in handy.
+
+```sh
+$ column -t snp_report.txt
+```
+
+Giving you this output:
+
+```
```
Comment (jhwz) [inline content/learn/helical-tools/geno.md:92]: Don't include the full output, they can run the command. Just the first few lines is great!

File: content/learn/helical-tools/geno.md [hunk 8]
```diff
@@ -0,0 +1,175 @@
...
+SNP00050  0.33  4    5    5    6    0.20  0.25  0.25  0.30
+```
+
+This file shows the number of alleles, their proportions and the minor allele frequency for each snp.
+
+The genotype_report.txt file holds the same information as above, except it's based on each genotype instead, and has a callrate column,
+which is a measure of the proportion of individuals that have a valid allele (not an NC).
+
+```
+ID      CallRate  #AA  #AB  #BB  #NC  %AA   %AB   %BB   %NC
```
Comment (jhwz) [inline content/learn/helical-tools/geno.md:152]: Same here, no need for the full output

File: content/learn/helical-tools/geno.md [hunk 9]
```diff
@@ -0,0 +1,175 @@
...
+SNP00046  0.33  3    7    6    4    0.15  0.35  0.30  0.20
+SNP00047  0.42  6    5    6    3    0.30  0.25  0.30  0.15
+SNP00048  0.47  6    7    5    2    0.30  0.35  0.25  0.10
+SNP00049  0.30  4    4    5    7    0.20  0.20  0.25  0.35
+SNP00050  0.33  4    5    5    6    0.20  0.25  0.25  0.30
+```
+
+This file shows the number of alleles, their proportions and the minor allele frequency for each snp.
+
+The genotype_report.txt file holds the same information as above, except it's based on each genotype instead, and has a callrate column,
```
Comment (jhwz) [inline content/learn/helical-tools/geno.md:148]:
```suggestion
The `genotype_report.txt` file holds the same information as above, except it's based on each genotype instead, and has a callrate column,
```

**Comment by jhwz** [comment]
Good start, your style is about right. A few comments there with some feedback but I think you have the right idea

File: content/learn/helical-tools/geno.md [hunk 10]
```diff
@@ -0,0 +1,119 @@
...
+
+# Geno
+
+## Getting Started
+
+This tutorial will walk you through the process of running geno commands. We will begin with analysing a geno file and understanding the different files that are created. To begin, download the geno file below.
+
+### genotype File
+
+A file that holds the genotype data for animals. It is arranged in a matrix where each row represents a different animal and each column represents a different marker (SNP). The first column will be an identifier for the animal.
```
Comment (jhwz) [inline content/learn/helical-tools/geno.md:17]:
```suggestion
A file that holds the genotype data for animals. It is arranged in a matrix where each row represents a different animal and each column represents a different marker (SNP). The first column is the animal identifier.
```

**Review by jhwz — COMMENTED** [review]
Looks pretty good, I have a couple more minor suggestions!

File: content/learn/helical-tools/geno.md [hunk 11]
```diff
@@ -0,0 +1,119 @@
...
+$ column -t snp_report.txt
+```
+
+Here's the first few lines of the output to be expected:
+
+```
+SnpID     MAF   #AA  #AB  #BB  #NC  %AA   %AB   %BB   %NC
+SNP00001  0.33  4    5    4    7    0.20  0.25  0.20  0.35
+SNP00002  0.53  7    7    4    2    0.35  0.35  0.20  0.10
+SNP00003  0.28  3    5    7    5    0.15  0.25  0.35  0.25
```
[the remark this answers is not in the document]
Comment (BlaviButcher) [reply content/learn/helical-tools/geno.md:102]: Makes sense

**Pull request merged**